src/ledger/test/LedgerTestUtils.cpp

```diff
@@ -535,5 +535,13 @@ generateLedgerHeadersForCheckpoint(
     makeValid(res, firstLedger, state);
     return res;
 }
+
+UpgradeType
```

Comment:
I don't think this really belongs in LedgerTestUtils, but I couldn't see a more logical header to but it in.

Reply:
Any reason not to keep it where it used to be?

Reply:
Since several different test files use this function now, I thought it would make more sense to put it in a utils header instead of `TxTest.h`.